### ISSUE TITLE
[Bert/TF1.13] WARNING:tensorflow:It seems that global step (tf.train.get_global_step) has not been increase

### ISSUE THREAD

**Issue by BattashB** [NVIDIA/DeepLearningExamples#624]
I'm trying to run bert base pretraining using this cmdline
` bash scripts/run_pretraining_lamb.sh 64 8 8 6e-5 4e-5 fp32 true 8 2000 200 7820 100 128 512 base 
`
I'm keep getting the following warning:
`WARNING:tensorflow:It seems that global step (tf.train.get_global_step) has not been increased. Current value (could be stable): 5 vs previous value: 5. You could increase the global step by passing tf.train.get_global_step() to Optimizer.apply_gradients or Optimizer.minimize.`

You may say that it's not harmful, but the training just keep going with this warning , without any advancement in training.
I saw in a different issue, that you said it's because the loss is becoming NaN , if that so , why is it becoming NaN. I 'm following your exact guidelines.

Please don't ask me to change to fp16, since it does not help, I checked.
**To Reproduce**
I did exactly as the quick start guide said.

**Expected behavior**
To train and converge....

**Environment**
 nvcr.io/nvidia/tensorflow:19.06-py3
8x Tesla V100-SXM2-32GB
TF1.13
Driver Version: 418.152.00   CUDA Version: 10.1     

I attached the full log of the training.
[tf_bert_pretraining_lamb_base_fp32_gbs165536_gbs232768.200727144018.log](https://github.com/NVIDIA/DeepLearningExamples/files/4986693/tf_bert_pretraining_lamb_base_fp32_gbs165536_gbs232768.200727144018.log)

**Comment by swethmandava** [comment]
It must accumulate for 128 steps before it updates the global step. Hence, the warning in itself is not harmful. I kicked off a run (attached partial log), it seems to be running ok.
[tf_bert_pretraining_lamb_base_fp16_gbs165536_gbs232768.200728193701.log](https://github.com/NVIDIA/DeepLearningExamples/files/4991124/tf_bert_pretraining_lamb_base_fp16_gbs165536_gbs232768.200728193701.log)

**Comment by BattashB** [comment]
Ok Thanks, I'll give it another try, and if I'll have troubles, Ill reopen the issue.
Thanks.
